<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="SSC1-W217" name="tutorial3.exception.TestRuntime" tests="1" failures="0" timestamp="6 Jun 2014 08:33:17 GMT" time="0.009" errors="0">
  <testcase name="divisionWithException" time="0.009" classname="tutorial3.exception.TestRuntime"/>
</testsuite> <!-- tutorial3.exception.TestRuntime -->
